pkg/synthetictests/alerts.go

```diff
@@ -12,14 +13,14 @@ import (
 	"github.com/openshift/origin/pkg/synthetictests/allowedalerts"
 )
 
-func testAlerts(events monitorapi.Intervals, restConfig *rest.Config, duration time.Duration) []*junitapi.JUnitTestCase {
+func testAlerts(events monitorapi.Intervals, restConfig *rest.Config, duration time.Duration, m *monitor.Monitor) []*junitapi.JUnitTestCase {
```

Comment:
can you pass the monitorapi.ResourceState (I don't recall the exact name) instead of pulling a monitor dep.